src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -352,7 +352,9 @@
             Sequence(Ref("ParameterNameSegment"), Ref("DatatypeSegment")),
         ),
         Sequence(
-            OneOf("DEFAULT", Ref("EqualsSegment")), Ref("LiteralGrammar"), optional=True
+            OneOf("DEFAULT", Ref("EqualsSegment")),
+            Ref("ExpressionSegment"),
```

Comment:
Should the `ExpressionSegment` be inside the `OneOf()`? Otherwise, I'm confused, because `LiteralGrammar` was inside. 🤔

Reply:
Look again, it wasn’t.

The `DEFAULT` and Equals were, as need one of them (e.g. `DEFAULT 1` or `= 1`).

And the whole Sequence is optional, as it still is.

Reply:
Ah yes, you're right. Sometimes Black annoys me -- it's so mechanical about what it places on the same vs different lines, it confuses me sometimes.